src/ig/IGExporter.ts

```diff
@@ -1313,7 +1360,8 @@ export class IGExporter {
       'files.',
       '',
       'NOTE: This file does not currently list the FHIR resources and examples generated from .fsh files. It only',
-      'lists those files generated from project configs or the contents in the ig-data folder.',
+      'lists those files generated from project configs or the contents in the',
+      `${path.basename(this.igDataPath)} folder.`,
```

Comment:
Is this intentionally adding a newline here?  This is an example of the message w/ master:
```
NOTE: This file does not currently list the FHIR resources and examples generated from .fsh files. It only
lists those files generated from project configs or the contents in the ig-data folder.
```
and this is in this PR:
```
NOTE: This file does not currently list the FHIR resources and examples generated from .fsh files. It only
lists those files generated from project configs or the contents in the
ig-data folder.
```

Reply:
No. I think this was one of the places I tried to fight prettier and put it on a new line, but I'll revert it.